app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @user.present? %>
```

Comment:
(style) avoid using instance variables in partials.

If we're sharing this partial for all forms it won't always be `user` (or `@user`). Consider inlining this until it needs to be separated. We can then use a general variable name (e.g. devise uses `resource`, Katalyst generally uses `subject`):

```
# shared/_error_messages.html.erb
<% if subject.errors.any? %>
  <%# render the error block... %>
<% end %>

# users/new.html.erb
<%= render "shared/error_messages", subject: @user %>
```
By passing the instance variable we can avoid extra the `@user.present?` check in the errors.